src/transformers/integrations.py

```diff
@@ -1181,7 +1176,7 @@ def get_run(cls, trainer):
                 return callback.run
 
         # TODO: NPT-12189 Update the exception
-        raise Exception("Trainer has any NeptuneCallback configured")
+        raise Exception("The trainer doesn't have a NeptuneCallback configured.")
```

Comment:
Wasn't sure about the wording here -- is this what it's supposed to mean?

Reply:
It's perfect now :wink:

Reply:
Haha great, thanks